packages/core/src/InstanceSettings.ts

```diff
@@ -61,6 +72,16 @@ export class InstanceSettings {
 	 */
 	instanceRole: InstanceRole = 'unset';
 
+	/**
+	 * Transient ID of this n8n instance, for scaling mode.
+	 * Reset on restart. Do not confuse with `instanceId`.
+	 *
+	 * @example 'main-bnxa1riryKUNHtln'
+	 * @example 'worker-nDJR0FnSd2Vf6DB5'
+	 * @example 'webhook-jxQ7AO8IzxEtfW1F'
+	 */
+	readonly hostId: string;
```

Comment:
should we setup [`member-ordering` linting rule](https://typescript-eslint.io/rules/member-ordering/) to have consistent placement of properties in classes across the code?

Reply:
I've always found this rule unnecessarily restrictive. I find it more important to be able to group fields and methods semantically rather than alphabetically.